requirements_test.txt

```diff
@@ -11,3 +11,4 @@ pytest-xdist~=2.5
 # Type packages for mypy
 types-pkg_resources==0.1.3
 tox>=3
+towncrier~=21.9
```

Comment:
I would include the dependency here so that contributors can actually use the `towncrier create` command.